src/App.tsx

```diff
@@ -1,73 +1,109 @@
+import { red, blue, deepOrange, grey, green } from '@mui/material/colors';
+
 import './App.css';
-import CarForm from './components/car/car_form';
-import { useState } from 'react';
-import CarList from './components/car/car_list';
-import { Car } from './types';
-import MaterialUISwitch from './components/materialUISwitch'; 
-import { createTheme, ThemeProvider } from '@mui/material';
-import {  useEffect } from 'react';
 
 function App() {
-  const [cars, setCars] = useState<Car[]>([]);
-  const [darkMode, setDarkMode] = useState<boolean>(false);
+  const [Motorcycles, setMotorcycles] = useState<Motorcycle[]>([]);
+  const [theme, setTheme] = useState('light');
 
-  const addCar = (car: Car) => {
-    setCars([...cars, car]);
+  const addMotorcycle = (Motorcycle: Motorcycle) => {
+    setMotorcycles([...Motorcycles, Motorcycle]);
   };
 
-  const toggleDarkMode = () => {
-    setDarkMode(!darkMode);
+  const handleThemeChange = () => {
+    if (theme === 'light') {
+      setTheme('dark');
+    } else {
+      setTheme('light');
+    }
   };
-
-  const theme = createTheme({
-    palette: {
-      mode: darkMode ? 'dark' : 'light',
-      primary: {
-        main: '#646bf3',
-      },
-      secondary: {
-        main: '#fff',
-      },
-      background: {
-        default: darkMode ? '#101113' : '#fff',
+  const lightTheme = responsiveFontSizes(
+    createTheme({
+      palette: {
+        mode: 'light',
+        primary: blue,
+        secondary: {
+          main: green[500],
+        },
+        divider: blue[100],
+        background: {
+          default: '#eee',
+          paper: '#fff',
+        },
+        text: {
+          primary: '#000',
+          secondary: grey[500],
+        },
       },
-      text: {
-        primary: darkMode ? '#fefefe' : '#101113',
+    })
+  );
+  
+  const darkTheme = responsiveFontSizes(
+    createTheme({
+      palette: {
+        mode: 'dark',
+        primary: deepOrange,
+        secondary: {
+          main: green[700],
+        },
+        divider: deepOrange[700],
+        background: {
+          default: '#333',
+          paper: '#222',
+        },
+        text: {
+          primary: '#fff',
+          secondary: grey[200],
+        },
       },
-    },
-  });
-
-  useEffect(() => {
-    document.body.style.backgroundColor = theme.palette.background.default;
-    document.body.style.color = theme.palette.text.primary;
-  }, [theme]);
+    })
+  );
   
 
+  let selectedTheme;
+  if (theme === 'light') selectedTheme = lightTheme;
+  else selectedTheme = darkTheme;
+
   return (
-    <ThemeProvider theme={theme}>
-      <div className='app-container'>
-      <div className="theme-toggle" style={{ position: 'fixed', top: '20px', right: '20px' }}>
+    <ThemeProvider theme={selectedTheme}>
+      <Box
+        className={`app-container ${theme}`}
+        sx={{
+          backgroundColor: theme === 'light' ? '#fff' : '#222', 
+          padding: '30px', 
+        }}
```

Comment:
Since you are now using `MUI` components and `sx` prop, you don't need to use `px` as it considered as hardcoded value. Instead you should just write a value which will be automatically scaled by MUI.

For example if you write `padding: 1` it will equals to `padding: 8px` since default scale factor is 8.
You can also change the factor as you want, check this docs: https://mui.com/material-ui/customization/spacing/

Reply:
Please, make sure you are not using `px` anywhere else